qa-pipelines/set-pipeline

```diff
@@ -87,6 +87,11 @@ if ! test -r "${pipeline_file}" ; then
     exit 1
 fi
 
+alt_secrets_file="CONCOURSE_SECRETS_FILE_${pool//-/_}"
```

Comment:
The pool name will be something like "azure", "gke", etc. so the substitution here is unnecessary

I'm guessing you were allowing the secrets to be pulled from a file besides blackbox for debugging? But I'm not really sure what's happening here or if we need it. If we do, a comment and note in the relevant README section about alternate usages would be helpful.